keyserver/listener.go

```diff
@@ -0,0 +1,127 @@
+package keyserver
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net"
+	"time"
+
+	. "github.com/coniks-sys/coniks-go/protocol"
+)
+
+func (server *ConiksServer) listenForRequests(ln net.Listener, handler func(msg []byte) ([]byte, error)) {
```

Comment:
I didn't notice that the you removed `<-server.stop` from `ln.Accept()`. Sorry for the noise.
Is this your suggestion? 2e182419b8b77ece38e599f4cc4a49b2c04dbf70